lib/ansible/modules/cloud/azure/azure_rm_devtestlabvirtualmachine_facts.py

```diff
@@ -0,0 +1,307 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2019 Zim Kalinowski, (@zikalino)
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_devtestlabvirtualmachine_facts
+version_added: "2.8"
+short_description: Get Azure DevTest Lab Virtual Machine facts.
+description:
+    - Get facts of Azure DevTest Lab Virtual Machine.
+
+options:
+    resource_group:
+        description:
+            - The name of the resource group.
+        required: True
+    lab_name:
+        description:
+            - The name of the lab.
+        required: True
+    name:
+        description:
+            - The name of the virtual machine.
+    tags:
```

Comment:
Xiang mentioned that DTL tags is hiding, could you pls figure out how it works regarding adding tags and filter by tags

Reply:
I just rechecked that, and:
- when dtl vm is created tags are properly assigned to that dtl vm
- dtl  vm has "compute_id" property which is the id of real underlying vm
- real vm + additional resources are created in that separate resource group

so this fact implementation is correct and provides enough information to locate actual vm.

Reply:
so filtering by tag  on DTL vm is same as normal VM?

Reply:
yes